### DISCUSSION TITLE
Meshing only certain polyhedras for a given tesselation

### DISCUSSION THREAD

**Discussion by AlexS1986** [neperfepx/neper#857] [Q&A] [answered]
Hello community,

My goal is to generate foam like meshes and meshes with pores. My plan is to generate a suitable tesselation and choose and instruct the mesh module to only mesh certain polyhedras. I thought that the `-meshpoly` parameter plus a suitable expression would do what I want to accomplish. However, most of my attempts have failed. 

I have tried 

```# Run Neper with specified options
neper -T -n 10 -reg 1  
# Run Neper with the generated tess file
neper -M  n10-id1.tess -order 1 -meshpoly "polynb<5" #all #"x<0.5"
```

The ```"x<5"``` is the only expression that worked -> only polyhedras with x<0.5 where meshed. However, I would prefer to specify the polyhedra numbers to be meshed. ```"polynb<5"``` (and other attempts) is just ignored and everything is meshed. **How could I exclude a certain polyhedra (or multiple polyhedras) from the mesh?**

Thank you guys in advance! You are awesome.

**Comment by rquey** [comment, marked as answer]
Here would be the syntax:
```
neper -M  n10-id1.tess -order 1 -meshpoly "id!=5" -nset none -faset none
```
(sorry for the delayed response).
